### ISSUE TITLE
Aria attributes, roles, etc

### ISSUE THREAD

**Issue by rootical** [anvk/a11yAccordion#48]
Hello. I wonder why there're no aria-labels, aria-expanded, hidden, roles and other things necessary according to WAG and other a11y standards. 
So how comes it's accessible?

**Comment by anvk** [comment]
Hi @rootical 

To answer this question you should ask what "accessible" means? You can find and read about this topic everywhere online. This is just one of the online examples:

> The Web is fundamentally designed to work for all people, whatever their hardware, software, language, location, or ability. When the Web meets this goal, it is accessible to people with a diverse range of hearing, movement, sight, and cognitive ability.

> Web accessibility means that websites, tools, and technologies are designed and developed so that people with disabilities can use them. More specifically, people can: perceive, understand, navigate, and interact with the Web, contribute to the Web.

Taken from [www.w3.org](https://www.w3.org/WAI/fundamentals/accessibility-intro/)

The attributes you specified above could help you to make of your code accessible out of the box (thanks to the modern browser support) but they do not _necessarily_ guarantee that. Adding them to your code would not make magically things accessible, you still have to do extra work to ensure this.
If you ask a person (let's say with visual visual impairment) and tell them that your code has those attributes and hence it is _accessible_ - it would not make much sense for them. Your "work" is accessible when it works well with assistive technology OR **even better** let people with different range of abilities to test it (color blindness, visual impairment, etc).

Now going back to the **a11yAccordion**. Yes you are right there. This code does not have those  attributes you have specified. But at the same time were just in their experimental stage 3 years ago.
What is important is that **a11yAccordion** was tested using different assistive technologies (JAWS, Mac built-in text-to-speech) as well as it was tested by various users with a range of abilities. Furthermore, this component was also used in [ATutor](https://atutor.github.io/) which is LMS for students with various needs. ATutor in fact was written by the same developer who created [AChecker](https://achecker.ca/checker/index.php) mentioned as the tool to test your websites for accessibility on [https://www.ontario.ca/page/how-make-websites-accessible](https://www.ontario.ca/page/how-make-websites-accessible)

The only concern is that **a11yAccordion** was written and tested a while ago. I'm not sure how much browsers and assistive technology has changed since then. But then this code uses pretty standard DOM properties so I doubt that it would be much different.

@rootical I hope it answers your question.
